changes/1998.added

```diff
@@ -0,0 +1 @@
+Added DynamicFilterForm to list views.
```

Comment:
Let's add a section to the Release Overview in `docs/release-notes/version-1.5.md` calling this out - it's a flagship feature of this release.